build.proj

```diff
@@ -33,8 +33,10 @@
     <ExeExtension>.exe</ExeExtension>
     <ExeExtension Condition=" '$(OS)' != 'Windows_NT' "></ExeExtension>
     <DotnetStage0>$(Stage0Path)/dotnet$(ExeExtension)</DotnetStage0>
-
     <DotnetCliBuildDirectory>$(RepoRoot)/build_projects/dotnet-cli-build</DotnetCliBuildDirectory>
+    <OutputDirectory>$(RepoRoot)/artifacts</OutputDirectory>
+    <PackagesDir>$(RepoRoot)/.nuget</PackagesDir>
+    <ToolsDir>$(RepoRoot)/build_tools/</ToolsDir>
```

Comment:
Yes. We already have a BuildToolsDir that holds the same information. We just wanted to be explicit as to what tools dir it is.